spec/un_locode_spec.rb

```diff
@@ -94,10 +94,17 @@
       let!(:country) { UnLocode::Country.create name: 'NETHERLANDS', code: 'NL' }
       let!(:location) { UnLocode::Locode.create name: 'Venlo', city_code: 'VEN', country: country }
 
-      context 'with supported functions' do
+      context 'with supported functions and space in search' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.